task/sql-tasks.js

```diff
@@ -281,7 +441,29 @@ async function task_1_21(db) {
  * @return {array}
  */
 async function task_1_22(db) {
-    throw new Error("Not implemented");
+    let result = await db.query(`
+    SELECT DISTINCT
+		Customers.CompanyName as CompanyName,
+		Products.ProductName as ProductName,
+		OrderDetails.UnitPrice as PricePerItem
+	FROM Customers 
+		RIGHT JOIN Orders on Orders.CustomerID = Customers.CustomerID
+        RIGHT JOIN OrderDetails on Orders.OrderID = OrderDetails.OrderID

```

Comment:
` RIGHT JOIN` is redundant because we have no orderdetails without orders , just JOIN